### ISSUE TITLE
Laravel 9 installation

### ISSUE THREAD

**Issue by roleosalaprojects** [antonioribeiro/countries#191]
Using version ^0.8.0 for pragmarx/countries
./composer.json has been updated
Running composer update pragmarx/countries
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires pragmarx/countries ^0.8.0 -> satisfiable by pragmarx/countries[v0.8.0].
    - pragmarx/countries v0.8.0 requires psr/simple-cache ^2.0 -> found psr/simple-cache[2.0.0, 2.x-dev] but it conflicts with your root composer.json require (^3.0).

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.

**Comment by antonioribeiro** [comment]
The composer.json on your application is preventing pragmarx/countries to be installed.
But I'm just adding psr/simple-cache ^1.0 to the stack so it will be supported.

**Comment by antonioribeiro** [comment]
Done.

**Comment by roleosalaprojects** [comment]
Thank you very much!

**Comment by Muffinman** [comment]
Just a note that I've just tried this on Laravel 9 myself but I'm getting a very similar error:

```
$ composer require pragmarx/countries
Info from https://repo.packagist.org: #StandWithUkraine
Using version ^0.8.2 for pragmarx/countries
./composer.json has been updated
Running composer update pragmarx/countries
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires pragmarx/countries ^0.8.2 -> satisfiable by pragmarx/countries[v0.8.2].
    - pragmarx/countries v0.8.2 requires psr/simple-cache ^1.0|^2.0 -> found psr/simple-cache[1.0.0, 1.0.1, 2.0.0, 2.x-dev] but the package is fixed to 3.0.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
You can also try re-running composer require with an explicit version constraint, e.g. "composer require pragmarx/countries:*" to figure out if any version is installable, or "composer require pragmarx/countries:^2.1" if you know which you need.
```

```
$ composer why psr/simple-cache
laravel/framework  v9.5.1  requires  psr/simple-cache (^1.0|^2.0|^3.0)
```

Since Laravel itself does support psr/simple-cache ^2.0 the simplist solution would be to pin that version before attempting to install.
